services/validation-service.js

```diff
@@ -483,6 +612,18 @@ class ValidationService {
         this.validateParamType('paranetDescription', paranetDescription, 'string');
     }
 
+    validateParanetNodesAccessPolicy(paranetNodesAccessPolicy) {
+        this.validateRequiredParam('paranetNodesAccessPolicy', paranetNodesAccessPolicy);
+        this.validateParamType('paranetNodesAccessPolicy', paranetNodesAccessPolicy, 'number');
+        if (paranetNodesAccessPolicy < 0 || paranetNodesAccessPolicy > 1) throw Error(`Invalid nodes access policy: ${paranetNodesAccessPolicy}. Should be 0 for OPEN or 1 for CURATED`);
```

Comment:
Perhaps we can add options for Access Policies to constants. Something similar to what you added in the demo, then we can just check if value is present in `Object.values`